src/pages/public_registration/LogPanel.tsx

```diff
@@ -41,10 +56,39 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     cacheTime: 1_800_000, // 30 minutes
   });
 
+  const logs: LogList = [...tickets];
+
+  if (registration.publishedDate) {
+    const registrationPublished: NonTicketLogEntry = {
+      modifiedDate: registration.publishedDate,
+      description: t('registration.status.PUBLISHED_METADATA'),
+    };
+    logs.push(registrationPublished);
```

Comment:
lurte først på om det er et problem å ha `.push()` rett i render-funkjsjonen som her, da det vil gjøres for hver render og lista kan potensielt dupliseres for hver render. Men etter en liten sjekk mener jeg det burde gå fint pga `const logs: LogList = [...tickets];`, sånn at man alltid begynner med en ny blank state 🤔

Reply:
Ja, det var det som var tanken bak spread-operatoren. Har blitt brent av duplisering altfor mange ganger...